homeassistant/components/lutron/light.py

```diff
@@ -72,7 +73,9 @@ def turn_on(self, **kwargs: Any) -> None:
             if ATTR_BRIGHTNESS in kwargs and self._lutron_device.is_dimmable:
                 brightness = kwargs[ATTR_BRIGHTNESS]
             elif self._prev_brightness == 0:
-                brightness = 255 / 2
+                brightness = self.platform.config_entry.options.get(
```

Comment:
(oh lol I missed this one).

We shouldn't get the config entry via `self.platform`, please just pass it as a parameter (or just pass the default dimmer level as parameter)